src/platform/windows.rs

```diff
@@ -78,6 +78,18 @@ pub trait WindowExtWindows {
     /// The pointer will become invalid when the native window was destroyed.
     fn hwnd(&self) -> *mut libc::c_void;
 
+    /// Enables or disables mouse and keyboard input to the specified window or control.
+    /// When input is disabled, the window does not receive input such as mouse clicks and key presses.
+    /// When input is enabled, the window receives all input.
+    ///
+    /// A window must be enabled before it can be activated.
+    /// For example, if an application is displaying a modeless dialog box and has disabled its
+    /// main window, the application must enable the main window before destroying the dialog box.
+    /// Otherwise, another window will receive the keyboard focus and be activated.
+    /// If a child window is disabled, it is ignored when the system tries to determine which
+    /// window should receive mouse messages.
+    fn set_enable(&self, enabled: bool);
```

Comment:
First of all it's awesome that you added such a thorough documentation.

Although, I would say that it may be beneficial to replace "main window" with "owner window" so that the terminology is more consistent.

Furthermore, adding a [link](https://doc.rust-lang.org/rustdoc/linking-to-items-by-name.html) to the `with_owner_window` function would be very helpful.